### PR TITLE
Update Avatar Icons On Releases Page

### PR DESCRIPTION
**Pull request by ayesha-omarali**
Switched out older component for AvatarList component on the releases page
[Resolves this ticket.](https://getsentry.atlassian.net/projects/APP/issues/APP-264?filter=myopenissues)

Before: sentry/project/releases overview
screen shot 2018-05-24 at 2 52 13 pm
After:
image

Before: sentry/project/releases
screen shot 2018-05-24 at 2 52 33 pm
After:
image

### CODE REVIEW

File: src/sentry/static/sentry/app/components/releaseStats.jsx [hunk 1]
```diff
@@ -33,15 +32,7 @@ const ReleaseStats = createReactClass({
       <div className="release-stats">
         <ReleaseSummaryHeading>{releaseSummary}</ReleaseSummaryHeading>
         <div className="avatar-grid">
```
Comment (lynnagara) [inline src/sentry/static/sentry/app/components/releaseStats.jsx]: We won't need this anymore, we should delete it as well

**Review by lynnagara — COMMENTED** [review]
Looks good, what do you think about making this say e.g. "3 other authors" as opposed to "3 other users".

**Comment by ckj** [comment]
😍

**Comment by benvinegar** [comment]
Awesome.

Aside: It'd be cool if avatar list knew how to expand itself to fit the available space.

**Review by lynnagara — APPROVED** [review]
🥇

**Pull request merged**